tests/test_future_py3.py

```diff
@@ -0,0 +1,31 @@
+import asyncio
+
+from pykka.compat import await_keyword
+
+from tests import has_async_await
+
+
+def _run_async(coroutine):
+    loop = asyncio.get_event_loop()
+    f = asyncio.ensure_future(coroutine, loop=loop)
+    return loop.run_until_complete(f)
+
+
+@has_async_await
+def test_future_supports_await_syntax(future):
+    @asyncio.coroutine
+    def _get_value():
+        return await_keyword(future)
+
+    future.set(1)
+    assert _run_async(_get_value()) == 1
+
+
+def test_future_supports_yield_from_syntax(future):
+    @asyncio.coroutine
+    def _get_value():
+        val = yield from future
+        return val
+
+    future.set(1)
+    assert _run_async(_get_value()) == 1
```

Comment:
No need to underscore prefix these two functions as they are part of the test suite and not the module anyway, and thus will never be mistaken as a public API.